boards/arm/nrf9161dk_nrf9161/nrf9161dk_nrf9161_common.dtsi

```diff
@@ -18,11 +18,11 @@
 	leds {
 		compatible = "gpio-leds";
 		led0: led_0 {
-			gpios = <&gpio0 0 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpio0 2 GPIO_ACTIVE_HIGH>;
 			label = "Green LED 1";
 		};
 		led1: led_1 {
-			gpios = <&gpio0 1 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpio0 3 GPIO_ACTIVE_HIGH>;
```

Comment:
You can't just "use unused GPIOs" here. How are they supposed to drive those LEDs?!

If the pins are to be used for a different purpose, you should use the GPIO expander for LEDs. See [nrf9161dk_leds_on_io_expander.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/368b0e0d16bfaa9bc833c76fbd7945cf5eabcbb3/boards/arm/nrf9161dk_nrf9161/dts/nrf9161dk_leds_on_io_expander.dtsi).

Reply:
> You can't just "use unused GPIOs" here. How are they supposed to drive those LEDs?!

Sure, this was my local hack to see if this solves Wi-Fi vs LED conflicts.

So, as the GPIO pins on board are fixed, do we have any other option to make both LEDs and Wi-Fi work? Or just disable LEDs when Wi-Fi is in use?